presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -207,6 +207,19 @@ public void testInsertIntoNotNullColumn()
         assertUpdate("DROP TABLE test_insert_not_null");
     }
 
+    @Test
+    public void testColumnComment()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.test_column_comment (col1 bigint COMMENT 'test comment', col2 bigint COMMENT '', col3 bigint)");
+
+        assertQuery(
+                "SELECT column_name, comment FROM information_schema.columns WHERE table_schema = 'tpch' AND table_name = 'test_column_comment'",
+                "VALUES ('col1', 'test comment'), ('col2', null), ('col3', null)");
```

Comment:
Why is col2's comment null and not `''`?

Reply:
Thanks, make sure to add this information in the code comment where `emptyToNull` is actually used

Reply:
after second thought i think that:
if comment in mysql is never null (fallbacks to `''`), we could preserve that. There is no reason for us to be "smarter".
This would also allow you not to override `getColumns` in `MySqlClient`.

(Of course, the distinction between `null` comment and `''` empty comment is not very practical and we may choose to normalize it -- let's keep is separate from this PR though and we would probably do this on engine level, not connector.)

Reply:
@findepi  If we preserve the actual empty comment in mysql, the result from SHOW CREATE TABLE will be awkward as below. What do you think about it? 
```sql
CREATE TABLE test (
 c1 int COMMENT '',
 c2 int COMMENT '',
 c3 int COMMENT '',
...
```

Reply:
i didn't think about this. You convinced me now.